config/sync/core.entity_form_display.taxonomy_term.basic_page_filters.default.yml

```diff
@@ -0,0 +1,46 @@
+uuid: 4afef6d5-9cc1-4e0c-8eb1-faa91bd1b95b
+langcode: en
+status: true
+dependencies:
+  config:
+    - taxonomy.vocabulary.basic_page_filters
+  module:
+    - path
+    - text
+id: taxonomy_term.basic_page_filters.default
+targetEntityType: taxonomy_term
+bundle: basic_page_filters
+mode: default
+content:
+  description:
```

Comment:
Let's add a description, as long as we are revising. Friendlier for site editors!

"Create filters for use with the Basic Page option for Filtered Lists paragraph"